sys/arduino/base.cpp

```diff
@@ -59,7 +59,7 @@ int digitalRead(int pin)
 
 void delay(unsigned long msec)
 {
-    xtimer_usleep(1000 * msec);
+    xtimer_usleep(US_PER_MS * msec);
```

Comment:
nit-picky: would be more consistent with other usages if you swap the order, i.e. `msec * US_PER_MS`

other then that, this looks good to go

Reply:
done and squashed